### DISCUSSION TITLE
Change jwt token cryptographic algorithm.

### DISCUSSION THREAD

**Discussion by yogendraapawar** [supabase/supabase#21751] [Questions]
The default cryptographic algorithm for access_token provided by supabase after signing in is HS256. How can I use different algorithm?

**Comment by GaryAustin1** [comment]
I don't believe that is possible.
Supabase gotrue repo would be where jwt signing is covered.
Two links that have comments from the devs about modifying the algorithm.
https://github.com/supabase/gotrue/pull/195
https://github.com/supabase/gotrue/pull/985
